analyses/molecular-subtyping-EWS/ews-subset/EWS_subtyping_path_dx_strings.json

```
@@ -0,0 +1,17 @@
+{
+    "exact_path_dx": [
+        "Ewings Sarcoma"
+    ],
+    "path_free_text_exact": [
+        "ewing's sarcoma",
+        "Ewings Sarcoma;Metastatic secondary tumors",
+        "Ewing Sarcoma",
+        "peripheral primitive neuroectodermal  tumor(ewing's sarcoma)",
+        "ewing sarcoma",
+        "medulloblastoma",
```

```suggestion
        "medulloblastoma",
```
we do not want to capture medullos here